### PR TITLE
Homework Task 4

### PR DESCRIPTION
**Pull request by Lilit-Lewonyan**

### CODE REVIEW

File: src/Task4.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+    }
+
+    /*N1: Write a function which will get parameter age
+     (int) and will return converted to days of live (int).
+     */
+    public static int getdaysoflife(int age) {
+        int yearscount366 = (age - (age % 4)) / 4;
+        int days = yearscount366 * 366 + (age - yearscount366) * 365;
+        return days;
```
Comment (Karengrigoryan060) [inline src/Task4.java:19]: please, add comments which will explain what was written

File: src/Task4.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    }
+
+    /* N2 : Write a function which will get age and return
+     is the person allowed to enter the club
+     (allows for people older the 18)
+      */
+    public static boolean isallowedforclub(int age) {
+        boolean isallowed = age >= 18 ? true : false;
+        return isallowed;
```
Comment (Karengrigoryan060) [inline src/Task4.java:28]: you could also return age >= 18 ? true : false; without any new variable

File: src/Task4.java (hunk 1 above)
Comment (Lilit-Lewonyan) [reply src/Task4.java:19]: okay Now I will do

File: src/Task4.java (hunk 2 above)
Comment (Lilit-Lewonyan) [reply src/Task4.java]: Done

File: src/Task4.java (hunk 1 above)
Comment (Lilit-Lewonyan) [reply src/Task4.java:23]: Done

**Pull request merged**